app/code/Magento/NewRelicReporting/etc/di.xml

```diff
@@ -41,6 +41,16 @@
         </arguments>
     </type>
     <type name="Symfony\Component\Console\Command\Command">
-        <plugin name="newrelic-describe-commands"  type="Magento\NewRelicReporting\Plugin\CommandPlugin"/>
+        <plugin name="newrelic-describe-commands" type="Magento\NewRelicReporting\Plugin\CommandPlugin"/>
+    </type>
+    <type name="Magento\Framework\Profiler\Driver\Standard\Stat">
+        <plugin name="newrelic-describe-cronjobs" type="Magento\NewRelicReporting\Plugin\StatPlugin"/>
+    </type>
+    <type name="Magento\NewRelicReporting\Plugin\CommandPlugin">
+        <arguments>
+            <argument name="skipCommands" xsi:type="array">
+                <item xsi:type="string" name="cron_run">cron:run</item>
```

Comment:
Why it should be skipped?

Reply:
Because we measure what's inside, so testing the container could be misleading.